db/migrate/20230922131629_add_icon_to_activity_category.rb

```diff
@@ -0,0 +1,5 @@
+class AddIconToActivityCategory < ActiveRecord::Migration[6.0]
+  def change
+    add_column :activity_categories, :icon, :string, default: 'volleyball-ball'
```

Comment:
can you use `clipboard-check` as that's what we're using on the school timeline of activities.

Also, if we're adding this as a field, then we'll need to add a control for it on the activity category admin forms, can you include that too?